mediajson/__init__.py

```diff
@@ -27,15 +27,21 @@
 except AttributeError:
     json_exception = ValueError
 
+try:
+  basestring
```

Comment:
this should have 4 spaces, otherwise looks good

Reply:
done